onnxruntime/core/graph/graph_utils.h

```diff
@@ -41,6 +41,9 @@ bool IsGraphInput(const Graph& graph, const NodeArg* input);
 /** Checks if the given node has only constant inputs (initializers). */
 bool AllNodeInputsAreConstant(const Graph& graph, const Node& node);
 
+/** Check if the input identified by the index to the give node is constant */
+bool IsNodeInputConstant(const Graph& graph, const Node& node, int index);
```

Comment:
doesn't look like you're actually using this method? It's in the commented out code in "ConvAddFusion::SatisfyCondition" but doesn't appear to be anywhere else. Remove this method?

Reply:
sorry, see my other comments below. I think you do need this method, but the code you had that's calling it is commented out right now (and shouldn't be?)

Reply:
fixed